dev-libs/libbpf/libbpf-1.0.0.ebuild

```diff
@@ -34,6 +34,7 @@ src_configure() {
 	append-cflags -fPIC
 	tc-export CC AR
 	export LIBSUBDIR="$(get_libdir)"
+	export LIBDIR="${EPREFIX}/usr/$(get_libdir)"
```

Comment:
I'd git mv for these fixes, as it could affect a "successfully" installed package.